### ISSUE TITLE
Include answering DNS server information in database

### ISSUE THREAD

**Issue by satta** [DCSO/balboa#20]
At least for the FEVER input, the feeder receives the IP of the answering DNS server. It could be useful to be able to store and query these data in the database backend.

**Comment by magenbluten** [comment]
true. with all the other requests / issues i fear the current database design would need a overhaul.
